eng/Packages.Data.props

```diff
@@ -191,9 +191,10 @@
     <PackageReference Update="OpenAI" Version="2.2.0-beta.4" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Developer.MicrosoftPlaywrightTesting'))">
+  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Developer.MicrosoftPlaywrightTesting')) or $(MSBuildProjectName.StartsWith('Azure.Developer.Playwright'))">
```

Comment:
Keeping this for the time being, till the older package is deprecated and cleaned-up.